src/native/libs/build-native.sh

```diff
@@ -83,7 +83,7 @@ elif [[ "$__TargetOS" == wasi ]]; then
     export WASI_SDK_PATH="${WASI_SDK_PATH%/}/"
     export CLR_CC="$WASI_SDK_PATH"bin/clang
     export TARGET_BUILD_ARCH=wasm
-    __CMakeArgs="-DCLR_CMAKE_TARGET_OS=wasi -DCLR_CMAKE_TARGET_ARCH=wasm -DWASI_SDK_PREFIX=$WASI_SDK_PATH -DCMAKE_TOOLCHAIN_FILE=$WASI_SDK_PATH/share/cmake/wasi-sdk.cmake $__CMakeArgs"
+    __CMakeArgs="-DCLR_CMAKE_TARGET_OS=wasi -DCLR_CMAKE_TARGET_ARCH=wasm -DWASI_SDK_PREFIX=$WASI_SDK_PATH -DCMAKE_TOOLCHAIN_FILE=$__RepoRootDir/src/native/external/wasi-sdk-p2.cmake $__CMakeArgs"
```

Comment:
Maybe we also need this in `src\native\libs\build-native.cmd`

Reply:
`build-native.cmd` uses `gen-buildsys.cmd`, which has the corresponding change.